events/lightning-talks.md

```diff
@@ -0,0 +1,18 @@
+# Lightning Talks
+
+## What is LT?
+- An event where CTMs present about a topic chosen by them, be it tech-related or not.
+- Each presentation should be of around 15 min
+
+## Purpose of the Event
+- To motivate CTMs to enhance their speaking and presentation skills (After all, they would be the ones conducting upcoming workshops)
+- To encourage healthy discussion among CTMs.
+
+## Before the Event
+- The event can last for 4-6 hours, so time can be divided into 2 days accordingly.
+- Room (with AV support) must be booked for the event.
+
+## After the Event
+Upload the following in the [kossitkgp/slides](https://github.com/kossiitkgp/slides) repo
+- Recording of the event, if its recorded
```

Comment:
I believe this point should not be under `slides`. If recorded, the event should probably be uploaded to YT after discussion and approval on Slack.

Reply:
In `slides` repo, in the README file, we have inserted YT links. That's why mentioned that here. Should I remove those links as well?

Reply:
It's fine if the YT links are in slides repo, but it may not always be recorded or made public, and the way this is worded, it feels like the recording should be directly added to the repo XD.

Reply:
oh sorry, i'll improve that

Reply:
Is it fine now?

Reply:
i'm working on the recordings one, else is fine i guess